frontend/src/components/CreateReview/index.jsx

```diff
@@ -0,0 +1,36 @@
+import { PropTypes } from 'prop-types';
+
+import { Breadcrumb, ErrorBanner, FrySpinner, RestaurantHeader } from '../Common/';
+import ReviewForm from './ReviewForm';
+
+const propTypes = {
+    error: PropTypes.string.isRequired,
+    currentRestaurant: PropTypes.object.isRequired,
+    currentReview: PropTypes.object.isRequired,
+    updateCurrentReview: PropTypes.func.isRequired,
+    createReview: PropTypes.func.isRequired
+};
+
+const CreateReview = ({ error, currentRestaurant, currentReview, updateCurrentReview, createReview }) => {
+    if (!currentRestaurant) {
```

Comment:
Consider giving priority to errors from backend if available. e.g. if user types restaurant ID that doesn't exist into URL, this will load forever.